lerna.json

```diff
@@ -23,8 +23,7 @@
       "prefix": false,
       "loglevel": "silent",
       "stream": true,
-      "concurrency": 8,
-      "sort": false
```

Comment:
It is important to compile dependencies first, dependents second. A package cannot be consumed before its compiled, because `types` and `main` are pointing to files that don't exist yet.

Reply:
We should keep `"concurrency": 8,` as it applies to modules that can be built in parallel. For example, with the following deps:
```
a -> b -> d
   -> c -> d
```
Module `b` and `c` can be complied independently.

Reply:
I am keeping that setting, here is the full diff:

```diff
- "concurrency": 8,
- "sort": false
+ "concurrency": 8
```